bigbluebutton-html5/tests/webdriverio/wdio.conf.js

```diff
@@ -0,0 +1,19 @@
+exports.config = {
+    specs: ['tests/webdriverio/specs/**/*.spec.js'],
+    capabilities: [
+        {
+            browserName: 'chrome'
+        }
+    ],
+    baseUrl: 'http://localhost:8080',
```

Comment:
It is already dynamic. These are all just default values. You can pass any custom config:
```sh
$ npm run test -- --baseUrl=http://...
```
It is similar to how you pass an autofix option to the linter.